### PR TITLE
5.31.0 release notes tck results

### PR DESCRIPTION
**Pull request by cubastanley**

### CODE REVIEW

**Review by rdebusscher — COMMENTED** [review]
Looks good

File: docs/modules/ROOT/pages/jakartaee-certification/5.31.0/README.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+= Jakarta EE Platform 8 certification summaries
```
Comment (debbiehoffman) [inline docs/modules/ROOT/pages/jakartaee-certification/5.31.0/README.adoc:1]: Jakarta EE Platform 8 Certification Summaries

File: docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * MicroProfile 4.1
+
+=== Notes
+
+=== New Features
+* [FISH-5646] Add Client Certificate Validation API
+
+=== Bug Fixes
+* [FISH-5711] Fix Could Not Load Formatter Class Error in Payara Micro
+* [FISH-5701] Application does not deploy anymore on Payara Server Docker image
```
Comment (debbiehoffman) [inline docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc:16]: Application Does Not Deploy Anymore on Payara Server Docker Image

File: docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* [FISH-5695] ArrayIndexOutOfBoundsException when the ConfigProperty value ends in a dollar sign \[Community Contribution\]
```
Comment (debbiehoffman) [inline docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc:17]: ArrayIndexOutOfBoundsException When the ConfigProperty Value Ends in a Dollar Sign \[Community Contribution\]

File: docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+* [FISH-5693] Delimiter for configuring multiple KeyStores and TrustStores must use JVM Platform Delimiter
```
Comment (debbiehoffman) [inline docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc:18]: Delimiter for Configuring Multiple KeyStores and TrustStores Must Use JVM Platform Delimiter

File: docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+* [FISH-5691] Do not propagate non-transactional EM to managed tasks
```
Comment (debbiehoffman) [inline docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc:19]: Do Not Propagate Non-Transactional EM to Managed Tasks

File: docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+* [FISH-5690] Remove duplicate postInvoke handler invocation on failure
```
Comment (debbiehoffman) [inline docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc:20]: Remove Duplicate postInvoke Handler Invocation on Failure

- Note - maybe this is PostInvoke ? not sure if it's a name and has to stay lowercase

File: docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+* [FISH-5660] Deployment Failure on Payara Micro
+* [FISH-1058] Payara Micro - wrong ClassLoader with multiple apps
```
Comment (debbiehoffman) [inline docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc:22]: Payara Micro - Wrong ClassLoader with Multiple Apps

File: docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+* [FISH-966] Fix Asadmin stop-domain help text
```
Comment (debbiehoffman) [inline docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc:23]: Fix Asadmin Stop-Domain Help Text

File: docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+* [FISH-81] Fix Incorrect error "No valid EE environment for injection" for CDI reported for events
```
Comment (debbiehoffman) [inline docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc:24]: Fix Incorrect Error "No Valid EE Environment for Injection" for CDI Reported Events

File: docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+
+=== Component Upgrades
+* [FISH-5655] Update Jersey to 2.34
+
+=== Security Fixes
+* [FISH-5702] Improper Limitation of a Pathname to a Restricted Directory \('Path Traversal'\) when context root is /
```
Comment (debbiehoffman) [inline docs/modules/ROOT/pages/release-notes/release-notes-31-0.adoc:30]: Improper Limitation of a Pathname to a Restricted Directory \('Path Traversal'\) When Context Root Is /

**Review by debbiehoffman — CHANGES_REQUESTED** [review]

**Review by Pandrex247 — COMMENTED** [review]
~Ignore - wrong PR~

**Review by OndroMih — APPROVED** [review]

**Pull request merged**